### ISSUE TITLE
Broker doesn't accept new connections after 24h running?

### ISSUE THREAD

**Issue by JFRosado** [martin-ger/uMQTTBroker#64]
I'm using this library on a ESP8266 (i've tried wemos D1 mini, nodemcu, etc). The broker is running and also publishing a value to itself (topic /Home/Voltage). I've another esp8266 (let's call it board2) that publishes to the broker (topic /Home/Current) and subscribes the topic  /Home/Voltage.

After 24h (this has happen with any of the mentioned boards used to the broker), I'm unable to connect to the broker with any other client, although the client in board 2 is still connected to the broker and receiving the subscribed topic.

**Comment by martin-ger** [comment]
Did you use the setting "lwip Variant: 1.4 High Bandwidth" in the "Tools" menu?

**Comment by JFRosado** [comment]
Yes, I did. I checked that when i changed from wemos to nodemcu, because on the first try I wasn't sure if I had used it.

**Comment by abdasgup** [comment]
I'm running uMQTT on ESP8266 as broker and publisher. PubSubClient on another ESP8266 as subscriber and on another one as publisher. After 5-6 minutes, the broker stops accepting any connection and both PubSubClient keeps getting -4 (timeout).

Those ESP8266s are programmed to go into deepsleep for 2 minutes and they do disconnect their connection before going to sleep, which shows up on broker (when it works). I have also powered down and powered back up on those ESP8266 to see if it was the PubSubClient's fault. But nothing fixed this issue. Only solution is to reboot the ESP8266 on which uMQTT broker is running.

I have tried to turn on debug of this library by including #define MQTT_DEBUG_ON as the first line in the ino file (before even #include this library. But nothing shows up on serial monitor. So I have absolutely no idea what is causing this library to not accept any new connection. Additionally I have tried changing IwIP to "v2 Higher Bandwidth" and that made no difference.

As a side note, I do also have a web server running on the same device and it's accepting connection perfectly fine, even when this issue is occurring.
